Date: Mon, 26 Jul 2021 09:10:26 +0000
From: 636f64144411f5a5118f1c4789c8d54315f8505e Mon Sep 17 00:00:00 2001
Subject: [PATCH] RDKTV-9419: Playerinfo to include AUTO mode for sound settings
Source: COMCAST
License: GPLV2
Upstream-Status: Pending
Signed-off-by:: Vinod Jacob <vinod_jacob@comcast.com>

Reason for change: Comcast TV platforms let the user select "Auto" mode
which automatically detects the max capability of the connected AVR
and transcodes the incoming stream to match the max caps of the AVR
wherever possible.
Signed-off-by: Vinod Jacob vinod_jacob@comcast.com
---
 interfaces/IDolby.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/interfaces/IDolby.h b/interfaces/IDolby.h
index a3a2c1d..7d181e8 100644
--- a/interfaces/IDolby.h
+++ b/interfaces/IDolby.h
@@ -46,7 +46,8 @@ namespace Exchange {
                 MONO,
                 STEREO,
                 SURROUND,
-                PASSTHRU
+                PASSTHRU,
+                SOUNDMODE_AUTO
             };
 
             /* @event */
-- 
2.25.1

